IntentsExtension/Actions/CompareLists.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import Intents
+
+@MainActor
+final class CompareListsIntentHandler: NSObject, CompareListsIntentHandling {
+	func handle(intent: CompareListsIntent) async -> CompareListsIntentResponse {
+		let response = CompareListsIntentResponse(code: .success, userActivity: nil)
+
+		let list1 = intent.list1 ?? []
+		let list2 = intent.list2 ?? []
+		let inputFiles = list1 + list2
+
+		let set1 = Set(list1.map(\.data))
+		let set2 = Set(list2.map(\.data))
```

Comment:
Using Swift Set operation does have the side effect of removing duplicates as well. Does that seem too unexpected?

Reply:
The Set needed to be of the Data to be correctly comparable. It seems like `INFile` compare as reference types.

Reply:
> Using Swift Set operation does have the side effect of removing duplicates as well. Does that seem too unexpected?

I think it's fine, but should be documented in the intent description.